tests/integration/targets/mso_tenant/tasks/main.yml

```diff
@@ -211,6 +211,42 @@
     - nm_add_tenant2_again.current.userAssociations | length == 1
   when: mso_username == 'admin'
 
+# ADD TENANT WITH REMOTE USERS
+- name: Add tenant 3 (check_mode)
+  mso_tenant:
+    <<: *tenant_present
+    tenant: ansible_test3
+    display_name: null
+    remote_users:
+    - name: ansible_github_ci
+      login_domain: test_2
+    state: present
+  check_mode: true
+  register: cm_add_rmt_usr_tenant3
+
+- name: Verify cm_add_rmt_usr_tenant3
+  assert:
+    that:
+    - cm_add_rmt_usr_tenant3 is changed
+
+- name: Add tenant 3 (normal_mode)
+  mso_tenant:
+    <<: *tenant_present
+    tenant: ansible_test3
+    display_name: null
+    remote_users:
+    - name: ansible_github_ci
+      login_domain: test
+    state: present
+  check_mode: true
```

Comment:
This probably shouldn't be check mode: true

Reply:
changed.